pipelines/srp-pipeline.yml

```diff
@@ -0,0 +1,522 @@
+resource_types:
+  #### Secure Release Pipeline resource type
+  - name: build-metadata
+    type: registry-image
+    source:
+      repository: pcfopsmanager/build-metadata-resource
+
+############## Resources ################
+resources:
+  #### Secure Release Pipeline resources
+  - name: srp-helper-test
+    type: registry-image
+    source:
+      repository: harbor.dhaka.cf-app.com/srp/srp-helper-task
+      username: ((phoenix-osspi.srp-harbor-username))
+      password: ((phoenix-osspi.srp-harbor-password))
+
+  - name: build-metadata
+    type: build-metadata
+
+  - name: metric-store-ci
+    type: git
+    source:
+      uri: git@github.com:cloudfoundry/metric-store-ci
+      private_key: ((phoenix-osspi.svcboteos_github_private_key))
+      branch: srp-scan
+
+  - name: metric-store-develop
```

Comment:
Change the resource name to correspond with the branch name, instead of metric-store-develop you can use metric-store-release or metric-store-master. Up to you

Reply:
Done